.env.sample

```diff
@@ -0,0 +1,8 @@
+HETNZER_TOKEN=
+HETNZER_SSH_NAME=
+LOCAL_SSH_KEY=~/.ssh/key
+POSTGRES=
+ETH_WSS=
+ETH_CONTRACT=
+RAD_SUBGRAPH=
+RAD_RPC_URL=
```

Comment:
Git is warning you that there is no new line at the end of some of these files, we should fix this as unix-based tooling relies on newlines to work correctly.

Reply:
Fixed https://github.com/radicle-dev/radicle-cloud/pull/1/commits/8586d567c76a667264059e5842ddb0be97b5dfd6